pkg/runtime/runtime.go

```diff
@@ -6,6 +6,12 @@ import (
 	"github.com/elotl/itzo/pkg/metrics"
 )
 
+const (
+	ItzoRuntimeName   = "itzo"
+	PodmanRuntimeName = "podman"
+	AnkaRuntimeName   = "anka"
```

Comment:
In general I’m not a fan of strings as constants. Strings in golang are immutable, so in a way doing this adds a level of indirection.

Personally I would either remove these constants and just use the strings "itzo", "podman", and "anka" strings directly, or make these constants part of an enum like this:

```golang
const (
    ItzoRuntimeName   = iota
    PodmanRuntimeName
    AnkaRuntimeName
)
```

Feel free to ignore this advice, it’s nitpicking :-)